xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/src/main/resources/Mail/Translations.xml

```diff
@@ -63,9 +63,9 @@ Mail.SendMailConfigClass_username.hint=The username to connect to the SMTP serve
 Mail.SendMailConfigClass_password=Email Server Password
 Mail.SendMailConfigClass_password.hint=The password to connect to the SMTP server, only when authentication is required
 Mail.SendMailConfigClass_properties=Additional Properties
-Mail.SendMailConfigClass_properties.hint=The list of additional JavaMail properties to use (e.g. "mail.smtp.starttls.enable=true" for TLS)
+Mail.SendMailConfigClass_properties.hint=List of additional JavaMail properties to use (e.g. "mail.smtp.starttls.enable=true" for TLS)
```

Comment:
Not consistent with the other hints (`The password`, `The username`, `The port`, etc.). Please change it back.

Reply:
Done.